targets/TARGET_RENESAS/TARGET_RZ_A1XX/TARGET_GR_LYCHEE/trng_api_esp32.cpp

```diff
@@ -65,13 +65,18 @@ extern "C" int trng_get_bytes_esp32(uint8_t *output, size_t length, size_t *outp
         send_data[0] = 0;
         ret = mI2c.write(ESP32_I2C_ADDR, send_data, 1);
         if (ret == 0) {
-            mI2c.read(ESP32_I2C_ADDR, recv_data, sizeof(recv_data));
+            ret = mI2c.read(ESP32_I2C_ADDR, recv_data, sizeof(recv_data));
```

Comment:
Why are we maintaining an error counter for the `write` calls, but none for `read`?